### PR TITLE
Add block:get command - Closes #273

### LINKED ISSUE #273: Add core block:get command
### Description

Add `block:get` command, accepting `id` or `height` as an argument, to obtain a specified block, according to the following specification:

1. It should allow both id and height as an argument
2. It should show an error if the node is not running locally
3. It should show decoded information
4. It should support:

```
--data-path (-d / LISK_DATA_PATH)
  Specifies which data path the application should use
```

### Motivation

Allow individual blocks to be obtained by id or height from the local node.

### Acceptance Criteria

- Add unit tests
- Examples and description are clear and correct
- Check if block can be obtained by id
- Check if block can be obtained by height

### Additional Information

Related issues: #254

### PR DESCRIPTION
**Pull request by ishantiw**
### What was the problem?

This PR resolves #273 

### How was it solved?

- Add command `block:get`
- Add unit tests related to `block:get`

### How was it tested?

`npm t`,
`./bin/run block:get --id=`,
`./bin/run block:get --height=`

### CODE REVIEW

File: src/commands/block/get.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import BaseIPCCommand from '../../base_ipc';
+
+interface Args {
+	readonly arg: string;
+}
+export default class GetCommand extends BaseIPCCommand {
```
Comment (nazarhussain) [inline src/commands/block/get.ts:20]:
```suggestion
export default class GetBlockCommand extends BaseIPCCommand {
```

File: src/base_ipc.ts [hunk 2]
```diff
@@ -148,7 +148,7 @@ export default abstract class BaseIPCCommand extends Command {
 				}: {
 					header: Buffer;
 					payload: ReadonlyArray<Buffer>;
-				} = codec.decodeJSON(blockSchema, blockBuffer);
+				} = codec.decode(blockSchema, blockBuffer);
```
Comment (nazarhussain) [inline src/base_ipc.ts:151]: Why its different than rest of the decode functions?

Comment (ishantiw) [reply src/base_ipc.ts:151]: we are expecting `header` and `payload` to be a buffer which is used later in the code whereas `decodeJSON` returns a base64 string for these 2 values

Comment (ishantiw) [reply src/base_ipc.ts:151]: Also, we have done the same in `base_plugin.ts` in `lisk-sdk`

File: src/commands/block/get.ts (hunk 1 above)
Comment (ishantiw) [reply src/commands/block/get.ts:20]: I wanted to change it to `GetBlockCommand` but then I used only `GetCommand` to be consistent with `account/get.ts`. I'm fine either way

**Pull request merged**